managed/devops/bin/yb_platform_util_py3.py

```diff
@@ -0,0 +1,483 @@
+            "status": "failed",
+            "error": "Universe with " + universe_name + " is not found."
+        }
+        handle_response(response)
+
+
+def get_universe_details_by_uuid(base_url, customer_uuid, auth_uuid, universe_uuid, base_dir):
+    """
+    Get universe details from UUID and store it in json after formatting it.
+
+    :param base_url: base url of the platform back end.
+    :param customer_uuid: UUID of the customer.
+    :param auth_uuid: authentication token for the customer.
+    :param universe_uuid: UUID of the universe to take a json data from.
+    :param base_dir: base directory in which the json file should be stored.
+    :return: None
+    """
+    universe = get_universe_by_uuid(base_url, customer_uuid, auth_uuid, universe_uuid)
+    if universe:
+        configure_json = create_universe_config(universe, universe["name"])
+        file = base_dir + '/' + universe["name"] + ".json"
+        with open(file, 'w') as file_obj:
+            json.dump(configure_json, file_obj)
+        response = {
+            "data": "Detail of universe has been saved to " + str(file),
+            "status": "success",
+            "error": ""
+        }
+        handle_response(response)
+    else:
+        response = {
+            "data": "Universe details not found",
+            "status": "failed",
+            "error": "Universe details not found"
+        }
+        handle_response(response)
+
+@exception_handling
+def create_universe_from_config(universe_config, base_url, customer_uuid, auth_uuid):
+    """
+    Create the universe from universe config data by calling universe POST API.
+
+    :param universe_config: universe config data.
+    :param base_url: base url of the platform back end.
+    :param customer_uuid: UUID of the customer.
+    :param auth_uuid: authentication token for the customer.
+    :return: None
+    """
+    universe_create_url = base_url + "/api/v1/customers/" + customer_uuid + "/universes"
+    response = call_api(universe_create_url, auth_uuid, universe_config)
+    universe_json = json.loads(response.read())
+    task_id = universe_json['taskUUID']
+    response = {"data": task_id, "status": "success", "error": ""}
+    print(response)
+
+
+def create_universe(base_url, customer_uuid, auth_uuid, input_file, universe_name=""):
+    """
+    Create the universe using the json and provided universe name.
+
+    :param base_url: base url of the platform back end.
+    :param customer_uuid: UUID of the customer.
+    :param auth_uuid: authentication token for the customer.
+    :param input_file: Directory of the stored universe json data.
+    :param universe_name: Name of the new universe to be created.
+    :return: None
+    """
+    configure_json = modify_universe_config(input_file, universe_name)
+    universe_config_json = post_universe_config(configure_json, base_url, customer_uuid, auth_uuid)
+    # add the missing fields in the config json.
```

Comment:
So it's just config generating ?

Reply:
Here, We are first creating configuration then and then calling create universe API.